.github/workflows/deploy-mainnet.yml

```diff
@@ -132,16 +132,13 @@ jobs:
           done
 
       - name: GIT | Commit changes to aleph-apps repository.
-        uses: EndBug/add-and-commit@v5.1.0
+        uses: EndBug/add-and-commit@v9.1.1
         with:
           author_name: AlephZero Automation
           author_email: alephzero@10clouds.com
           message: "Updating Mainnet docker image tag for release: ${{ steps.vars.outputs.branch }}"
           add: "*.yaml"
           cwd: "aleph-apps"
-          branch: mainnet
```

Comment:
Just to make sure – this is now not necessary, because the checkout step picks the branch and that's enough?

Reply:
I got the same question like @timorl

Reply:
yes @timorl , that's correct.  If you check the temporary workflow that has been created to test this, branch was taken from `actions/checkout@v3`.  There is no `branch` input in the latest version of `EndBug/add-and-commit`.